docs/blog/newsletter-july-2024.rst

```diff
@@ -0,0 +1,136 @@
+:og:image: https://www.writethedocs.org/_static/logo-opengraph.png
+
+.. post:: July 02, 2024
+  :tags: newsletter
+
+#####################################
+Write the Docs Newsletter – July 2024
+#####################################
+
+Cheers, documentarians! This week has seen a cooling off of temperatures after a heat wave across Europe. Joni Mitchell sang that you don't know what you got till it's gone, and it seems the same is true for things you don't want. I hope you are all able to avoid any unwelcome extremes in your lives, whatever forms they make take.
+
+In conference news, our Atlantic conference is coming up in September, so `get a ticket </conf/atlantic/2024/tickets/>`__ while you still can. Write the Docs Australia has `announced its call for proposals along with ticket sales </conf/australia/2024/news/announcing-cfp-tickets/>`__, so start getting your ideas in now. And if you're looking farther into the future, the Portland conference has `official dates </conf/portland/2025/news/welcome/>`__, May 4--6, 2025, so mark the time in your calendar.
```

Comment:
```suggestion
In conference news, our Atlantic conference is coming up in September, so `get a ticket </conf/atlantic/2024/tickets/>`__ while you still can. Write the Docs Australia has `announced its call for proposals along with ticket sales </conf/australia/2024/news/announcing-cfp-tickets/>`__, so start getting your ideas in now. And if you're looking farther into the future, the Portland conference has `official dates </conf/portland/2025/news/welcome/>`__, May 4-6, 2025, so mark the time in your calendar.
```

Reply:
The doubling here was intentional as it leads to an en dash in the published site (rather than a hyphen).